README.md

```diff
@@ -18,18 +18,15 @@
 ```shell
 bin/dev setup
 ```
-5. Start python and mongo containers.
```

Comment:
`docker-compose run` alleviates the need to do this. Instead you can just launch the ad hoc container for the task you want to run.